src/overlays/actors/ovl_En_Reeba/z_en_reeba.c

```diff
@@ -232,17 +243,17 @@ void func_80AE5054(EnReeba* this, PlayState* play) {
             }
 
             if (this->isBig) {
```

Comment:
Bump on this suggested change (isBig vs type != small), was it not done on purpose?

Reply:
I'm concerned that by no longer treating it as a bool, the code may be more cluttered. It seems only to check for a 0 or non-zero value. The "Reeba Boss Appears" log also suggests this field was likely solely used as a bool determining if it was the big purple Leever or small green one.

Reply:
imo its not cluttering anything. actors having a type variable and comparing aginst a type enum is a common construction. it only works out here cause the leever happens to have 2 types (0 and 1). You can imagine a modding scenario where more types are added (and it would work just fine with this variable)

that print message doesnt say anything about the variable and its type/how it was treated. its just to let the devs know that a leever that is big has spawned :D

Reply:
another point of conflict comes from a `[note from another PR](https://github.com/zeldaret/oot/pull/1431#discussion_r1020726133)` - wherein it's recommended to treat the int as a bool if it still matches. Thus far, this seems to be the case, and, IMO, `if (this->isBig)` seems to have as much legibility and conveys as much information as `if (this->leeverType == LEEVER_TYPE_BIG)` without being as verbose.

Reply:
that comment isnt really relevant to this situation. that one is pretty clearly a boolean.
This one is not as cut and dry. It *can* be treated as a boolean yes, but it also can just as well be a type variable (which, in the case of this actor, only has two types, 0 and 1)

Given that "type" is such a common pattern among all actors for this kind of thing, treating this value as a boolean is not really preferred.

Reply:
to be clear, something matching as `if (var)` doesnt mean it was a boolean originally. pretty much any `if (var != 0)` can match as `if (var)`. That comment just specifies that when we DO treat it as a boolean, it should be changed to that format.

Our point here is that we dont want to treat this as a boolean